public/editor/scripts/editor/js/fc/project-rename.js

```diff
@@ -132,15 +133,21 @@ define(function(require) {
         }
         editingComplete(context);
         analytics.event("ProjectRenamed");
+
+        var publisher = new Publisher();
```

Comment:
You shouldn't create a new instance of the publisher here. You already have access to a publisher instance in `this` (or out here it's `context`) so you should use that instead.